src/main/navigation/navigation_fixedwing.c

```diff
@@ -433,6 +442,14 @@ void applyFixedWingPitchRollThrottleController(navigationFSMStateFlags_t navStat
         uint16_t correctedThrottleValue = constrain(navConfig()->fw.cruise_throttle + throttleCorrection, navConfig()->fw.min_throttle, navConfig()->fw.max_throttle);
         rcCommand[THROTTLE] = constrain(correctedThrottleValue, motorConfig()->minthrottle, motorConfig()->maxthrottle);
     }
+
+    /*
+     * Then altitude is below landing slowdown min. altitude, set throttle to min_throttle
+     * TODO refactor conditions in this metod if logic is proven to be correct
+     */
+     if (sensors(SENSOR_BARO) && posControl.actualState.pos.V.Z < navConfig()->general.land_slowdown_minalt) {
```

Comment:
`sensors(SENSOR_BARO)` shouldn't be used explicitly. We may have only GPS altitude. You can check for `posControl.flags.hasValidAltitudeSensor` flag.